File: v2/java/src/main/java/io/swagger/client/model/Article.java
package io.swagger.client.model;


import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Article  {
  
  private String id = null;
  private String createdAt = null;
  private String title = null;
  private String body = null;
  private Boolean isResponsive = null;
  private Boolean isProxyNeeded = null;
  private Boolean isMobileProxyNeeded = null;
  private Boolean needsSanitize = null;
  private String proxyUrl = null;
  private String topic = null;
  private String suitabilityScore = null;
  private String summary = null;
  private String imagePath = null;
  private String imageUrl = null;
  private Integer imageHeight = null;
  private Integer imageWidth = null;
  private String logoUrl = null;
  private String squareLogoUrl = null;
  private String url = null;
  private String externalUrl = null;
  private String externalUrlSource = null;
  private String externalUrlType = null;

  
  /**
   **/
  @ApiModelProperty(required = true, value = "")
  @JsonProperty("_id")
  public String getId() {
    return id;
  }
  public void setId(String id) {
    this.id = id;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("created_at")
  public String getCreatedAt() {
    return createdAt;
  }
  public void setCreatedAt(String createdAt) {
    this.createdAt = createdAt;
  }

  
  /**
   **/
  @ApiModelProperty(required = true, value = "")
  @JsonProperty("title")
  public String getTitle() {
    return title;
  }
  public void setTitle(String title) {
    this.title = title;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("body")
  public String getBody() {
    return body;
  }
  public void setBody(String body) {
    this.body = body;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("is_responsive")
  public Boolean getIsResponsive() {
    return isResponsive;
  }
  public void setIsResponsive(Boolean isResponsive) {
    this.isResponsive = isResponsive;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("is_proxy_needed")
  public Boolean getIsProxyNeeded() {
    return isProxyNeeded;
  }
  public void setIsProxyNeeded(Boolean isProxyNeeded) {
    this.isProxyNeeded = isProxyNeeded;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("is_mobile_proxy_needed")
  public Boolean getIsMobileProxyNeeded() {
    return isMobileProxyNeeded;
  }
  public void setIsMobileProxyNeeded(Boolean isMobileProxyNeeded) {
    this.isMobileProxyNeeded = isMobileProxyNeeded;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("needs_sanitize")
  public Boolean getNeedsSanitize() {
    return needsSanitize;
  }
  public void setNeedsSanitize(Boolean needsSanitize) {
    this.needsSanitize = needsSanitize;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("proxy_url")
  public String getProxyUrl() {
    return proxyUrl;
  }
  public void setProxyUrl(String proxyUrl) {
    this.proxyUrl = proxyUrl;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("topic")
  public String getTopic() {
    return topic;
  }
  public void setTopic(String topic) {
    this.topic = topic;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("suitability_score")
  public String getSuitabilityScore() {
    return suitabilityScore;
  }
  public void setSuitabilityScore(String suitabilityScore) {
    this.suitabilityScore = suitabilityScore;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("summary")
  public String getSummary() {
    return summary;
  }
  public void setSummary(String summary) {
    this.summary = summary;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("image_path")
  public String getImagePath() {
    return imagePath;
  }
  public void setImagePath(String imagePath) {
    this.imagePath = imagePath;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("image_url")
  public String getImageUrl() {
    return imageUrl;
  }
  public void setImageUrl(String imageUrl) {
    this.imageUrl = imageUrl;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("image_height")
  public Integer getImageHeight() {
    return imageHeight;
  }
  public void setImageHeight(Integer imageHeight) {
    this.imageHeight = imageHeight;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("image_width")
  public Integer getImageWidth() {
    return imageWidth;
  }
  public void setImageWidth(Integer imageWidth) {
    this.imageWidth = imageWidth;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("logo_url")
  public String getLogoUrl() {
    return logoUrl;
  }
  public void setLogoUrl(String logoUrl) {
    this.logoUrl = logoUrl;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("square_logo_url")
  public String getSquareLogoUrl() {
    return squareLogoUrl;
  }
  public void setSquareLogoUrl(String squareLogoUrl) {
    this.squareLogoUrl = squareLogoUrl;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("url")
  public String getUrl() {
    return url;
  }
  public void setUrl(String url) {
    this.url = url;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("external_url")
  public String getExternalUrl() {
    return externalUrl;
  }
  public void setExternalUrl(String externalUrl) {
    this.externalUrl = externalUrl;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("external_url_source")
  public String getExternalUrlSource() {
    return externalUrlSource;
  }
  public void setExternalUrlSource(String externalUrlSource) {
    this.externalUrlSource = externalUrlSource;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("external_url_type")
  public String getExternalUrlType() {
    return externalUrlType;
  }
  public void setExternalUrlType(String externalUrlType) {
    this.externalUrlType = externalUrlType;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Article {\n");
    
    sb.append("  id: ").append(id).append("\n");
    sb.append("  createdAt: ").append(createdAt).append("\n");
    sb.append("  title: ").append(title).append("\n");
    sb.append("  body: ").append(body).append("\n");
    sb.append("  isResponsive: ").append(isResponsive).append("\n");
    sb.append("  isProxyNeeded: ").append(isProxyNeeded).append("\n");
    sb.append("  isMobileProxyNeeded: ").append(isMobileProxyNeeded).append("\n");
    sb.append("  needsSanitize: ").append(needsSanitize).append("\n");
    sb.append("  proxyUrl: ").append(proxyUrl).append("\n");
    sb.append("  topic: ").append(topic).append("\n");
    sb.append("  suitabilityScore: ").append(suitabilityScore).append("\n");
    sb.append("  summary: ").append(summary).append("\n");
    sb.append("  imagePath: ").append(imagePath).append("\n");
    sb.append("  imageUrl: ").append(imageUrl).append("\n");
    sb.append("  imageHeight: ").append(imageHeight).append("\n");
    sb.append("  imageWidth: ").append(imageWidth).append("\n");
    sb.append("  logoUrl: ").append(logoUrl).append("\n");
    sb.append("  squareLogoUrl: ").append(squareLogoUrl).append("\n");
    sb.append("  url: ").append(url).append("\n");
    sb.append("  externalUrl: ").append(externalUrl).append("\n");
    sb.append("  externalUrlSource: ").append(externalUrlSource).append("\n");
    sb.append("  externalUrlType: ").append(externalUrlType).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: v2/csharp/src/main/csharp/vestorly/Model/Article.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace vestorly.model {
  [DataContract]
  public class Article {
    
    
    [DataMember(Name="_id", EmitDefaultValue=false)]
    public string id { get; set; }

    
    
    [DataMember(Name="created_at", EmitDefaultValue=false)]
    public string createdAt { get; set; }

    
    
    [DataMember(Name="title", EmitDefaultValue=false)]
    public string title { get; set; }

    
    
    [DataMember(Name="body", EmitDefaultValue=false)]
    public string body { get; set; }

    
    
    [DataMember(Name="is_responsive", EmitDefaultValue=false)]
    public bool? isResponsive { get; set; }

    
    
    [DataMember(Name="is_proxy_needed", EmitDefaultValue=false)]
    public bool? isProxyNeeded { get; set; }

    
    
    [DataMember(Name="is_mobile_proxy_needed", EmitDefaultValue=false)]
    public bool? isMobileProxyNeeded { get; set; }

    
    
    [DataMember(Name="needs_sanitize", EmitDefaultValue=false)]
    public bool? needsSanitize { get; set; }

    
    
    [DataMember(Name="proxy_url", EmitDefaultValue=false)]
    public string proxyUrl { get; set; }

    
    
    [DataMember(Name="topic", EmitDefaultValue=false)]
    public string topic { get; set; }

    
    
    [DataMember(Name="suitability_score", EmitDefaultValue=false)]
    public string suitabilityScore { get; set; }

    
    
    [DataMember(Name="summary", EmitDefaultValue=false)]
    public string summary { get; set; }

    
    
    [DataMember(Name="image_path", EmitDefaultValue=false)]
    public string imagePath { get; set; }

    
    
    [DataMember(Name="image_url", EmitDefaultValue=false)]
    public string imageUrl { get; set; }

    
    
    [DataMember(Name="image_height", EmitDefaultValue=false)]
    public int? imageHeight { get; set; }

    
    
    [DataMember(Name="image_width", EmitDefaultValue=false)]
    public int? imageWidth { get; set; }

    
    
    [DataMember(Name="logo_url", EmitDefaultValue=false)]
    public string logoUrl { get; set; }

    
    
    [DataMember(Name="square_logo_url", EmitDefaultValue=false)]
    public string squareLogoUrl { get; set; }

    
    
    [DataMember(Name="url", EmitDefaultValue=false)]
    public string url { get; set; }

    
    
    [DataMember(Name="external_url", EmitDefaultValue=false)]
    public string externalUrl { get; set; }

    
    
    [DataMember(Name="external_url_source", EmitDefaultValue=false)]
    public string externalUrlSource { get; set; }

    
    
    [DataMember(Name="external_url_type", EmitDefaultValue=false)]
    public string externalUrlType { get; set; }

    
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Article {\n");
      
      sb.Append("  id: ").Append(id).Append("\n");
      
      sb.Append("  createdAt: ").Append(createdAt).Append("\n");
      
      sb.Append("  title: ").Append(title).Append("\n");
      
      sb.Append("  body: ").Append(body).Append("\n");
      
      sb.Append("  isResponsive: ").Append(isResponsive).Append("\n");
      
      sb.Append("  isProxyNeeded: ").Append(isProxyNeeded).Append("\n");
      
      sb.Append("  isMobileProxyNeeded: ").Append(isMobileProxyNeeded).Append("\n");
      
      sb.Append("  needsSanitize: ").Append(needsSanitize).Append("\n");
      
      sb.Append("  proxyUrl: ").Append(proxyUrl).Append("\n");
      
      sb.Append("  topic: ").Append(topic).Append("\n");
      
      sb.Append("  suitabilityScore: ").Append(suitabilityScore).Append("\n");
      
      sb.Append("  summary: ").Append(summary).Append("\n");
      
      sb.Append("  imagePath: ").Append(imagePath).Append("\n");
      
      sb.Append("  imageUrl: ").Append(imageUrl).Append("\n");
      
      sb.Append("  imageHeight: ").Append(imageHeight).Append("\n");
      
      sb.Append("  imageWidth: ").Append(imageWidth).Append("\n");
      
      sb.Append("  logoUrl: ").Append(logoUrl).Append("\n");
      
      sb.Append("  squareLogoUrl: ").Append(squareLogoUrl).Append("\n");
      
      sb.Append("  url: ").Append(url).Append("\n");
      
      sb.Append("  externalUrl: ").Append(externalUrl).Append("\n");
      
      sb.Append("  externalUrlSource: ").Append(externalUrlSource).Append("\n");
      
      sb.Append("  externalUrlType: ").Append(externalUrlType).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}
